packages/vkui/src/components/HorizontalCellShowMore/HorizontalCellShowMore.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Icon28ChevronRightCircle } from '@vkontakte/icons';
+import { classNames } from '@vkontakte/vkjs';
+import { HasRef, HasRootRef, LiteralUnion } from '../../types';
+import type { ImageBaseSize } from '../ImageBase/ImageBase';
+import { Tappable, TappableProps } from '../Tappable/Tappable';
+import { Subhead } from '../Typography/Subhead/Subhead';
+import styles from './HorizontalCellShowMore.module.css';
+
+export interface HorizontalCellShowMoreProps
+  extends Omit<TappableProps, 'getRootRef' | 'size'>,
+    HasRef<HTMLElement>,
+    HasRootRef<HTMLDivElement> {
+  /**
+   * Задаёт высоту компонента. Должeн соответствовать размеру картинок
+   * внутри соседних HorizontalCell компонентов.
+   *
+   * Используйте размеры заданные дизайн-системой `16 | 20 | 24 | 28 | 32 | 36 | 40 | 44 | 48 | 56 | 64 | 72 | 80 | 88 | 96`.
+   *
+   * > ⚠️ Использование кастомного размера – это пограничный кейс.
+   * Игнорируется если `size='s'`.
+   */
+  height?: LiteralUnion<ImageBaseSize, number>;
+  /**
+   * Задаёт размер компонента. Аналогичен такому же пропу у [HorizontalCell](https://vkcom.github.io/VKUI/#/HorizontalCellShowMore?id=props).
+   * Должен соответствовать значению пропа `size` соседних HorizontalCell.
+   */
+  size?: 's' | 'm' | 'l';
+  /**
+   * Предназначен для отрисовки текста.
+   * По умолчанию для `size='s'` содержит текст `Все` для других размеров `Показать все`.
+   * */
+  children?: TappableProps['children'];
+  /**
+   * Позволяет компенсировать особый правый отступ у предшевствующего последнего элементa HorizontalCell,
+   * в том случае, если тот элемент последний в родителе.
+   * Если HorizontalCellShowMore находится на одном уровне с остальными HorizontalCell, то этот проп использовать не нужно.
+   */
+  compensateLastCellIndent?: boolean;
+}
+
+export const HorizontalCellShowMore = ({
+  className,
+  style,
+  getRef,
+  getRootRef,
+  compensateLastCellIndent,
+  height,
+  size = 's',
+  children: childrenProp,
+  ...restProps
+}: HorizontalCellShowMoreProps) => {
+  const children = childrenProp || (size === 's' ? 'Все' : 'Показать все');
```

Comment:
```suggestion
  children = size === 's' ? 'Все' : 'Показать все',
  ...restProps
}: HorizontalCellShowMoreProps) => {
```

так в доке будет выводится значение по умолчанию, что в очередной раз покажет, что при `size="s"` другой текст

<img width="320" src="https://github.com/VKCOM/VKUI/assets/5850354/99429f52-cd87-4652-8df2-4078a945b424" /> <img width="320" src="https://github.com/VKCOM/VKUI/assets/5850354/2ab0b678-b142-458f-a0ab-21f321d0b219" />